.github/workflows/cockpit-container-update.yml

```diff
@@ -0,0 +1,31 @@
+name: cockpit-container-update
+on:
+  schedule:
+    - cron: '0 2 * * 4'
+  # can be run manually on https://github.com/cockpit-project/cockpit-machines/actions
+  workflow_dispatch:
+jobs:
+  cockpit-container-update:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+      statuses: write
+    steps:
+      - name: Set up dependencies
+        run: |
+          sudo apt update
+          sudo apt install -y make
+
+      - name: Set up configuration and secrets
+        run: |
+          printf '[user]\n\tname = Cockpit Project\n\temail=cockpituous@gmail.com\n' > ~/.gitconfig
+          echo '${{ secrets.GITHUB_TOKEN }}' > ~/.config/github-token
+
+      - name: Clone repository
+        uses: actions/checkout@v4
+
+      - name: Run cockpit-container-update
+        run: |
+          make bots
+          bots/cockpit-container-update
```

Comment:
It was renamed in https://github.com/cockpit-project/bots/pull/6095 , can  you please follow suit and also rename the workflow and job name?

Reply:
Did anyone test it on a fork?

Reply:
And it doesn't work at all.... https://github.com/jelly/cockpit-machines/actions/runs/8329606250/job/22792320551

Reply:
And now it does https://github.com/jelly/cockpit-machines/actions/runs/8330370703/job/22794851154